content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_clusters.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
```

Comment:
@matobet 
can you please add a spec for this method and all the other methods.
https://github.com/ManageIQ/manageiq-content/blob/master/spec/content/automate/ManageIQ/AutomationManagement/AnsibleTower/Operations/Methods.class/__methods__/available_credentials_spec.rb